src/sandstorm/web-session.capnp

```diff
@@ -111,6 +147,48 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
     encoding @2 :Text;  # Content-Encoding header (optional).
   }
 
+  struct PropfindContent {
```

Comment:
You know, maybe what we really want here is a single method called `dav()`, whose first parameter is an enum specifying the specific method. Otherwise we have a lot of repeated code which seems pretty redundant.

OPTIONS should still be handled specifically, though.